docs/configuration/monitoring.md

```diff
@@ -52,6 +71,18 @@ engine:
       enabled: true
 ```
 
+Apply this configuration by issuing an helm upgrade while passing these additional values.
+To do so append `--values monitoring-values.yaml --recreate-pods` to the command used to install
+codacy provided in [this documentation](../install.md).
+
+```sh
+helm upgrade (...options used to install codacy...) --values monitoring-values.yaml --recreate-pods
+```
+
+Grafana will be running in the same domain as Codacy, on port 30080, for instance
```

Comment:
How does this happen? We are not configuring the ingress for this, right?

Reply:
Yeah, it was a nodeport, but that's messy. I'll remove it for now, and then use codacy ingress when we add it to the chart.